scripts/fast-reboot

```diff
@@ -130,41 +130,36 @@ function parseOptions()
     done
 }
 
-function common_clear()
+function clear_boot()
 {
+    # common_clear
     debug "${REBOOT_TYPE} failure ($?) cleanup ..."
 
     /sbin/kexec -u || /bin/true
 
     teardown_control_plane_assistant
-}
-
-function clear_fast_boot()
-{
-    common_clear
-
-    sonic-db-cli STATE_DB DEL "FAST_REBOOT|system" &>/dev/null || /bin/true
-}
-
-function clear_warm_boot()
-{
-    common_clear
 
+    #clear_warm_boot
     result=$(timeout 10s config warm_restart disable; res=$?; if [[ $res == 124 ]]; then echo timeout; else echo "code ($res)"; fi) || /bin/true
     debug "Cancel warm-reboot: ${result}"
 
     TIMESTAMP=$(date +%Y%m%d-%H%M%S)
     if [[ -f ${WARM_DIR}/${REDIS_FILE} ]]; then
         mv -f ${WARM_DIR}/${REDIS_FILE} ${WARM_DIR}/${REDIS_FILE}.${TIMESTAMP} || /bin/true
     fi
+
+    #clear_fast_boot
+    if [[ "$REBOOT_TYPE" = "fast-reboot" ]]; then
+        sonic-db-cli STATE_DB DEL "FAST_REBOOT|system" &>/dev/null || /bin/true
+    fi
 }
 
 function init_warm_reboot_states()
 {
     # If the current running instance was booted up with warm reboot. Then
     # the current DB contents will likely mark warm reboot is done.
     # Clear these states so that the next boot up image won't get confused.
-    if [[ "$REBOOT_TYPE" = "warm-reboot" || "$REBOOT_TYPE" = "fastfast-reboot" ]]; then
+    if [[ "$REBOOT_TYPE" = "warm-reboot" || "$REBOOT_TYPE" = "fastfast-reboot" || "$REBOOT_TYPE" = "fast-reboot" ]]; then
```

Comment:
suggest to have comment here: why fast-reboot requires delete WARM_RESTART_TABLE|*.state

Reply:
This was introduced since this PR utilizes warm-reboot infrastructure for fast-reboot.